src/shim/runtime/v2/rune/v2/create.go

```diff
@@ -2,16 +2,21 @@ package v2
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"path/filepath"
 
 	"github.com/confidential-containers/enclave-cc/src/shim/runtime/v2/rune/config"
 	"github.com/confidential-containers/enclave-cc/src/shim/runtime/v2/rune/oci"
 	shimtypes "github.com/confidential-containers/enclave-cc/src/shim/runtime/v2/rune/types"
+	"github.com/containerd/containerd/api/types"
 	"github.com/containerd/containerd/runtime/v2/runc"
 	taskAPI "github.com/containerd/containerd/runtime/v2/task"
+	"github.com/sirupsen/logrus"
 )
 
+const ContainerBase = "/run/enclave-cc/app"
```

Comment:
it must be possible to mock this dir so that the unit tests can run:
```
    create_test.go:313: 
        	Error Trace:	create_test.go:313
        	Error:      	"mkdir /run/enclave-cc: permission denied" does not contain "failed to mount rootfs component"
        	Test:       	TestCreateApp
    create_test.go:315: 
```

Reply:
hi, does `mock this dir` means to create the dir manually or else? Could you please `sudo go test` to run the unit tests?

Reply:
> hi, does `mock this dir` means to create the dir manually or else? Could you please `sudo go test` to run the unit tests?

for unit tests, I'm expecting them to create temporary directories for the tests to run and then have them to cleanup afterwards. Even if `sudo go test` worked, it's not a good idea for the tests to create some system directory and leave it there. Therefore, I was asking to change this so that for the tests, we have `ContainerBase` in some `<tmpdir>/run/enclave-cc/app`.